qamomile/core/circuit/circuit.py

```diff
@@ -134,6 +135,13 @@ class ThreeQubitGate(Gate):
     control2: int
     target: int
 
+@dataclasses.dataclass
+class ParametricExpGate(Gate):
+    """Parametric exponential gate class."""
```

Comment:
It is better to write the definition of the gate in the docstring.
Is this $e^{i\theta H}$ or $e^{-i\theta H}$?